CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 ### Grafana Mimir
 
+* [BUGFIX] Rules: improve error handling when querier is local to the ruler. #7567
```

Comment:
Must be in 2.12.0-rc.0 section as per process

Reply:
We have already discussed this in a slack thread, and we agreed this is the right approach.